Resources/ServerInfo/Guidebook/DeltaV/Rules/GameRules/2_Metagaming.xml

```diff
@@ -1,14 +1,130 @@
+
+  The fact that the nuke disk must be protected and could be used by a bad actor to try to destroy the station is not shielded.
+
+  The revealing condition for this shield is any of the following:
+  - discovering a blood red hardsuit
+  - discovering a nuclear operative's shuttle
+  - an operative name
+  - a War Ops announcement
+  - being a nuclear operative
+
+  ## Implants
+
+  The existence of implants used by the Syndicate is shielded. Implant checks cannot be performed unless there is adequate evidence or an authorized warrant, as outlined within Standard Operating Procedures.
+
+  The revealing condition for this shield is any of the following:
+  - discovering a non-NT implanter, used or unused
+  - discovering a non-NT implant box
+  - discovering use of a non-NT implant by anyone
+  - experiencing a situation where absolutely no other explanation is possible
+  - discovering an unlocked uplink
+
+  ## Chameleon Items
+
+  Chameleon items are shielded.
+
+  Being suspicious of an item being fake or stolen is not shielded, but testing items or calling them chameleon is covered by this shield. Testing clothing for chameleon status is always prohibited; items that are clearly chameleon can, however, be confiscated by Security.
+
+  The revealing condition for this shield is any of the following:
+  - seeing someone else cause any chameleon item to change
+  - finding holographic nanomachine fibers
+  - experiencing a situation where absolutely no other explanation is possible
+  - discovering an unlocked uplink
+
+  ## Stealth Items
+
+  The fact that an item can be something other than what its visual appearance and examine description indicate is shielded.
+
+  This shield protects stealth items, including protecting them from being tested.
+
+  The revealing condition for this shield is any of the following:
+  - seeing the item behave differently than the expected behavior for the item
+  - seeing the item used for its hidden purpose
+  - experiencing a situation where absolutely no other explanation is possible
+  - discovering an unlocked uplink
+
+  ## Bad Actor Objectives
+
+  The idea that Syndicate agents, or other bad actors, have certain goals they have been tasked with accomplishing on the station, ranging from sabotage, to theft, to murder is shielded.
+
+  The revealing condition for this shield is any of the following:
+   - being a Traitor, Space Ninja, Nuclear Operative, or another bad actor with goals.
+   - being told of an antagonist’s goals. Note that this only applies to the goals that have been revealed to you.
+
+  ## Uplinks
+
+  The fact that Syndicate agents may possess a program, referred to as an Uplink, used to receive equipment from the Syndicate is shielded. You have no way of confirming an Uplink exists within a PDA unless you see it yourself.
+
+  The revealing condition for this shield is any of the following:
+   - being a Security role
+   - being a Command role
+   - being a traitor
```

Comment:
Shouldn't finding an unlocked uplink or seeing someone use it also count?

Reply:
> unless you see it yourself.